### PR TITLE
feat(nav): Show dropdown hover

### LINKED ISSUE #12290: Expand dropdown on hover
The navbar dropdown should expand on hover. Here is the updated links JSON: https://gist.github.com/QuincyLarson/e5739f334bc72daf7cf77005099c2156

### PR DESCRIPTION
**Pull request by BerkeleyTrue**
closes #12290

### CODE REVIEW

File: common/app/components/Nav/Nav.jsx [hunk 1]
```diff
@@ -31,14 +31,17 @@ function handleNavLinkEvent(content) {
 }
 
 const propTypes = {
```
Comment (Bouncey) [inline common/app/components/Nav/Nav.jsx]: Finally you are starting to alphabetise things! :smile:

File: common/app/redux/actions.js [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import { Observable } from 'rx';
 import { createAction } from 'redux-actions';
 import types from './types';
+import noop from 'lodash/noop';
```
Comment (Bouncey) [inline common/app/redux/actions.js]: What is the value of loadash/noop over `() => {}`?

**Review by Bouncey — APPROVED** [review]

File: common/app/components/Nav/Nav.jsx (hunk 1 above)
Comment (BerkeleyTrue) [inline common/app/components/Nav/Nav.jsx]: yeah, now I just need to find a eslint rule to enforce this everywhere.

File: common/app/redux/actions.js (hunk 2 above)
Comment (BerkeleyTrue) [inline common/app/redux/actions.js]: You only need to define the noop once. Otherwise you are creating the noop several times throughout the app

**Comment by BerkeleyTrue** [comment]
@Bouncey If it works for you please merge

**Pull request merged**